src/core/src/basics/sample.cpp

```diff
@@ -169,7 +169,8 @@ bool Sample::load()
 {
 	// Will contain a bunch of metadata about the loaded sample.
 	SF_INFO sound_info;
-	
+	sound_info.format = 0;
```

Comment:
Would it hurt to zero whole struct, just in case? `SF_INFO sound_info = {0}`?

Reply:
I initially went with just 'format' because that makes it explicit that it's a specific requirement of the API (a very odd requirement, at that!) but initialising the whole thing does look cleaner and read better.